include/deal.II/lac/relaxation_block.h

```diff
@@ -210,11 +223,11 @@ class RelaxationBlock :
    * vectors). For the Jacobi step, the calling function must copy @p dst to
    * @p pref after this.
    */
-  template <typename number2>
+  template <class VECTOR>
```

Comment:
I think we've mostly standardized on `VectorType` as the name of the template argument, following @drwells earlier rounds of patches.